blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+
+That's problematic with Gather though, as the human interaction is much different in the Metaverse. 
+Ideally, we can answer the following questions through a **Metaverse Netiquette**. 
+Here are some ideas of questions and answers.
+
+**What are the signs not to approach someone?** <br>
+_Check the status message first and then the situation_
++ 🟢 _If the person sits in the kitchen, it's fine to approach them as this signals an informal working environment_
++ 🟠 _If the person sits at a desk, decide wisely as you still might interrupt them_
++ 🔴 _If it's red, you rather not interrupt_
+
+**Two people have a chat. Can I just bump into them?** <br>
++ 🟢 _If it's in the hallway, it's okay for sure_ 
++ 🟠 _In the kitchen it might be okay_
++ 🔴 _It's never okay if it's a meeting room_
+
+**I have a quick question. Write a message or find that person and speak to them?** <br>
+_This depends on the question as it would in real life_
++ 🟢 _Sensitive topic? Yes please, to prevent any misinterpretations in the text_
++ 🟠 _Urgent? Yes please, or find another adequate channel for the given topic_
++ 🔴 _Random question that has time? Send a text message for an asynchronous answer_
+
+**Do I need to start Gather every day?** <br>
+_Yes, of course. Otherwise, you can't participate in the Metaverse and people will miss you_
+
+**Can I close the Gather application?** <br>
+_Ideally, you keep it running, but if you have some client meeting through a different channel you might want to close it to prevent any interruptions_
+
+**We are physically in the office, but others are not. Can we merge reality with the Metaverse?** <br>
+_That's tricky because we are dealing with the crossing of two realities. Everyone or at least one person in the physical have to join with their avatar to create a bridge between the two worlds_
+
+There are probably many more questions to clarify. I think I will create a living netiquette for Satellytes in 2022.
+I hope that document will help people to adapt Gather more easily. 
+
+## Conclusion
+Satellytes did not start as a remote-only company more than three years ago. People enjoyed going to the office and talking to each other. That's why our folks had an actual pain when we were forced to work remotely. On the other side, everyone feels the power of remote work, that we want to embrace in the future. 
+
+The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I got a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with that experiment in 2022. 
```

Comment:
```suggestion
The Metaverse seems to be the glue we have been missing all the time. Gather is an excellent way to dive into the Metaverse. I became a big fan of Gather in the past weeks. I'm happily tinkering around with their [mapmaker](https://github.com/gathertown/mapmaking) and constantly improving our virtual space because I really want our folks to enjoy the Metaverse. Let's see where we are heading with that experiment in 2022. 
```

Reply:
thanks

Reply:
I think the parsing from github is broken 😂 The highlighted words are escaped and not displayed with the adequate highlight.

![Screen Shot 2021-12-22 at 15 53 47](https://user-images.githubusercontent.com/1701755/147111435-3855e63a-3309-487e-bec9-cbfe6d554b65.png)